### DISCUSSION TITLE
slow rendering of chat window

### DISCUSSION THREAD

**Discussion by alonsoac** [LiveHelperChat/livehelperchat#1988] [General]
Hi. This is a problem reported by several users on different servers and different computers. It seems a recent upgrade of Chrome and possibly Edge is causing problems with LHC. Many functions are very slow, like loading any page and even typing in message textarea. For example when typing one letter the onkeypress event can take 1 or more seconds to complete and so the letters take a long time to appear one by one while typing.
See video https://drive.google.com/file/d/1fOCyE04I9mO_6gAvf6lYIbDAhcXIEH_1/view?usp=sharing
This happens to me in Windows 11 with updated Chrome.  Older Chrome in Windows 8.1 is not affected. Firefox is not affected.  So really most users are affected as Chrome updates automatically.

**Comment by memotext-ahmad** [comment]
same issue for me

**Reply by remdex** [reply]
https://livehelperchat.com/chrome-116-bug-slow-back-office.-updated-2-681a.html

**Reply by remdex** [reply]
Chrome will fix this bug in next it's release. They confirmed it's their bug. You can either use firefox as edge within update have same bug. Or just update LHC as I did workaround for chrome bug...

**Comment by mike-miguel** [comment]
Hi,

After update from v4.06 to current v.426, I have a conflict in the dababase:
**Database structure check with official version
lh_abstract_survey - identifier index was not found
lh_chat - iwh_id index was not found, theme_id index was not found**

How I can proceed?
Thanks in advance,
![screenshot 1](https://github.com/LiveHelperChat/livehelperchat/assets/74366703/b9989362-5f2f-4776-82ae-8b996b37dcaf)

**Reply by remdex** [reply]
Try to click update database just again. If not try to execute these queries manually just.

**Reply by mike-miguel** [reply]
Thanks, but how I can execute these queris manually?

**Comment by remdex** [comment]
use phpmyadmin or any other database management tool.

**Reply by mike-miguel** [reply]
I can access to phpmyadmin, but I don't know the systaxis that I must write to execute the query. Can you point me at least the first one?
` ADD `identifier` varchar(50) NOT NULL, COMMENT='';
` ADD INDEX `identifier` (`identifier`);

Thanks in advance

**Reply by remdex** [reply]
Each row is a separate query... So execute one by one.

**Reply by mike-miguel** [reply]
Thanks a lot. But I have this error:
![screenshot 2](https://github.com/LiveHelperChat/livehelperchat/assets/74366703/7753b102-044c-4a9e-a21d-03f5ba553a4f)

(translated: #1118 - Row size too large. The maximum row size for the type of table used, not counting BLOBs, is 8126. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs.)

Thanks

**Reply by remdex** [reply]
That's your server issue...

Execute both queries at once.
```
SET GLOBAL innodb_strict_mode = 0;
alter table...
```

**Reply by mike-miguel** [reply]
Thanks! It works! Cheers!! 😎
